File: java/parser/src/main/java/gov/lbl/parser/domain/Import_clause.java
package gov.lbl.parser.domain;

public class Import_clause {
    private String identifier;
    private Name name;
    private Boolean dot_star;
    private Import_list import_list;
    private Comment comment;

    public Import_clause(String identifier, Name name, Boolean dot_star, Import_list import_list, Comment comment) {
        this.identifier  = identifier;
        this.name = name;
        this.dot_star = dot_star;
        this.import_list = import_list;
        this.comment = comment;
    }
}

File: jsParser/domain/Import_clause.js
class Import_clause {
  constructor (identifier, name, dot_star, import_list, comment) {
    identifier != null && identifier != '' ? this.identifier = identifier : ''
    name != null ? this.name = name : ''
    dot_star != null ? this.dot_star = dot_star : ''
    import_list != null ? this.import_list = import_list : ''
    comment != null ? this.comment = comment : ''
  }
}
exports.Import_clause = Import_clause
